common/routes/userRoutes.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+    
+    SIGNUP:{
+
+        http:{ path:'/signup', verb:"post" },
+        accepts:{ arg:'data', type:"object", http:{ source:"body" } },
+        returns:{ arg:"data", type:"object" } 
+    },
+
+    LOGIN:{
+
+        http:{ path:'/login', verb:'post' },
+        accepts:{ arg:'data', type:'object', http:{ source:'body' } },
+        returns:{ arg:'data', type:"string" }
+    },
+
+    SEND_OTP:{
```

Comment:
here, you just copy all the data in all the methods, need to create a function to pass the required values and that's it.